packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -90,6 +98,12 @@ export type UserStorageControllerEnableProfileSyncing =
 export type UserStorageControllerDisableProfileSyncing =
   ActionsObj['disableProfileSyncing'];
 
+export type UserStorageControllerStateChangeEvent = ControllerStateChangeEvent<
+  typeof controllerName,
+  UserStorageControllerState
+>;
+export type Events = UserStorageControllerStateChangeEvent;
```

Comment:
Can we rename this to UserStorageControllerEvents to align with the rest of the controllers ? The same would be for Actions?

Reply:
This is also due to how notifications intends to define subpath exports in https://github.com/MetaMask/core/pull/4604. Depending on the implementation, we may need to enforce a globally-unique, namespaced name for the Events type, but that will need to happen in its own PR along with fixes for all of the other types in the notifications packages.

- Previous discussion: https://github.com/MetaMask/core/pull/4573#discussion_r1697262249